charts/testkube-operator/templates/tests.testkube.io_tests.yaml

```diff
@@ -428,7 +428,12 @@ spec:
                           type: string
                       type: object
                     type:
-                      description: test type
+                      description: |
+                        type of sources a runner can get data from.
+                            string: String content (e.g. Postman JSON file).
+                            file-uri: content stored on the webserver.
+                            git-file: the file stored in the Git repo in the given path.
+                            git-dir: the entire git repo or git subdirectory (Testkube does a spatial checkout to limit traffic in the case of monorepos).
```

Comment:
actually, we also support `git` type, it automatically detects file or dirs

Reply:
thanks, I gave it a try into 12313d90a70804887a7c456dbb9210112fb16a9f does it properly captures current behavior ?

Reply:
internally, we already trreat everything as git. yes, the idea is to support monorepo as well

Reply:
thanks,

> BTW, is it intended that the sparse checkout leaves the /data/repo with root of the git repo and not the subdir as new root ?

 sorry I had missed the `workingDir` field which answers my question

```
                        workingDir:
                          description: if provided we checkout the whole repository
                            and run test from this directory
                          type: string
``